src/core/layout/qgslayout.h

```diff
@@ -303,14 +303,16 @@ class CORE_EXPORT QgsLayout : public QGraphicsScene, public QgsExpressionContext
     /**
      * Returns the topmost layout item at a specified \a position. Ignores paper items.
      * If \a ignoreLocked is set to TRUE any locked items will be ignored.
+     * \since QGIS 3.32 searchTolerance parameter was added, which can be used to specify a search tolerance in layout units
```

Comment:
Hmm, the sipify script isn't handling this well -- it's tagging the whole method as being added in 3.32:

```suggestion
     * Since QGIS 3.32 the \a searchTolerance parameter was added, which can be used to specify a search tolerance in layout units.
```